Dockerfile

```diff
@@ -2,33 +2,37 @@ FROM kylemanna/openvpn:latest
 
 ARG K8S_VERSION=v1.5.1
 
-ADD https://storage.googleapis.com/kubernetes-release/release/$K8S_VERSION/bin/linux/amd64/kubectl /usr/local/bin/kubectl
-RUN chmod +x /usr/local/bin/kubectl
-
 ARG GITHUB_TOKEN
 ARG REPO=cloudposse/github-pam
 ARG FILE=github-pam_linux_386
 ARG VERSION=0.10
+
 ARG PAM_SCRIPT_VERSION=1.1.8-1
 
 ADD rootfs /
 
-ADD https://gist.githubusercontent.com/goruha/dc4c5eca4d8322b19ff718d5e1510723/raw/4709cc7c794e1945aa84a3ed08d5a04de14fbda6/gh-dl-release /bin/gh-dl-release
+ADD https://storage.googleapis.com/kubernetes-release/release/$K8S_VERSION/bin/linux/amd64/kubectl /usr/local/bin/kubectl
+RUN chmod +x /usr/local/bin/kubectl
+
+ADD https://raw.githubusercontent.com/cloudposse/build-harness/master/scripts/gh-dl-release /bin/gh-dl-release
```

Comment:
At this point, I'd probably just have travis download the release using the build harness rather than complicate the `Dockerfile`; or at least I would do that in the future.